src/main/resources/deploy/cluster/cluster-with-db-loadbalancer/docker-compose-xld-ha-with-db-loadbalancer.yaml

```diff
@@ -0,0 +1,165 @@
+version: "3.4"
+
+services:
+  zoo1:
+    image: zookeeper:3.4
+    hostname: zoo1
+    container_name: zoo1
+    ports:
+      - 2191:2181
+    networks:
+      - {{DEPLOY_NETWORK_NAME}}
+    environment:
+      ZOO_MY_ID: 1
+      ZOO_SERVERS: server.1=0.0.0.0:2888:3888 server.2=zoo2:2888:3888 server.3=zoo3:2888:3888
+
+  zoo2:
+    image: zookeeper:3.4
+    hostname: zoo2
+    container_name: zoo2
+    ports:
+      - 2192:2181
+    networks:
+      - {{DEPLOY_NETWORK_NAME}}
+    environment:
+      ZOO_MY_ID: 2
+      ZOO_SERVERS: server.1=zoo1:2888:3888 server.2=0.0.0.0:2888:3888 server.3=zoo3:2888:3888
+    depends_on:
+      - zoo1
+
+  zoo3:
```

Comment:
The number of nodes and partitions have to be configurable.
I might want not 3 but 5 nodes.
Should be some simple way to specify it in config.
Like `partitions: 5`, and all this file is generated. Exposed ports have to be random, and the function used which identifies which port is free now. Check in the code, we have it.